Carts with no items cannot be submitted

djkeithly opened the pull request:
A third button state that is not interact able has been added to the shopping cart viewer when there is no item in cart Edge case of item = 0, directly press button was tested and it should be fine

When cart has an item:
![image](https://github.com/user-attachments/assets/b2295fc1-d904-4375-bca3-f0170a50b133)

When cart does not:
![image](https://github.com/user-attachments/assets/65b49d0c-f5a3-4c48-98b5-bef228b12609)

CrazEpic reviewed (approved):
Mainly checked out functionality, looks good

Commit:
Carts with no items cannot be submitted
A third button state that is uninteractable has been added to the shopping cart viewer when there is no item in cart
Edge case of item = 0, direclty press button was tested and it should be fine

CrazEpic committed:
change disabled buttons to gray instead of red

CrazEpic reviewed (approved):
awesome
